usefulUtilities/hiFiCalendar/entityServerScripts/hiFiCalendarServer.js

```diff
@@ -8,119 +8,348 @@
 
 
 (function() {
+    var INTERVAL_FREQUENCY_MS = 60000;
+    var EXPIRY_BUFFER = 300000;
+    var HOURS_PER_DAY = 24;
+    var NOON = 12;
+    var CHANNEL = "HiFi.Google.Calendar";
+    var GREEN_TEXT = [168, 255, 168];
+    var RED_TEXT = [255, 168, 168];
+    var GREEN = [125, 255, 125];
+    var RED = [255, 125, 125];
```

Comment:
could you make this and the above vars slightly more descriptive, please? maybe `SIGN_BACKGROUND_COLOR_INUSE` ?